components/bms/bms_master/Core/Inc/soc.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file soc.h
+ * @brief Header file that contains functions for calculating the state of charge 
+ *        for UBC Solar's V3 solar car, Brightside using interpolation and 
+ *        coloumb counting
+ * 
+ * 
+ * @date 2021/11/09
+ * @author Edward Ma, Forbes Choy 
+ */
+
+
+#ifndef __SOC_H
+#define __SOC_H
```

Comment:
Just for consistency with other modules, please name the include guard
```suggestion
#ifndef INC_SOC_H_
#define INC_SOC_H_
```
Remember to update the comment by the `#endif` as well